v2/emailpassword/nextjs/app-directory/next-steps.mdx

```diff
@@ -1,6 +1,6 @@
 ---
 id: next-steps
-title: 8. Next steps
+title: 7. Next steps
 hide_title: true
 ---
```

Comment:
not related to this file, but in the `setting-up-frontend` page, for custom UI, you need to explain that you need to create a component that has `use client` directive in it, and that component wraps their entire app (in layout.tsx) file. In the global scope in that component, you need to do:
```
if (typeof window !== "undefined") {
  SuperTokensWebJs.init(frontendConfig())
}
```

Reply:
done